src/client/components/insurance-information/AdditionalInformationSection.jsx

```diff
@@ -65,7 +65,7 @@ class AdditionalInformationSection extends React.Component {
               value={this.props.data.vaMedicalFacility}
               facilityState={this.props.data.facilityState}
               onValueChange={(update) => {this.props.onStateChange('vaMedicalFacility', update);}}/>
-          OR <a target="_blank" href="/facility-locator">Go to the VA Facility Locator</a>
+          OR <a target="_blank" href="http://www.va.gov/directory/guide/home.asp">Find locations with the VA Facility Locator</a>
```

Comment:
Hm do we need to use the va.gov facility locator if we have one of our own? I think we should use our facility locator tool going forward, unless there is strong reason not to?